cardano-crypto-class/src/Cardano/Crypto/VRF/Simple.hs

```diff
@@ -166,7 +160,8 @@ instance VRFAlgorithm SimpleVRF where
     let u = h' (toCBOR a) k
         y = h $ toCBOR a <> toCBOR u
         VerKeySimpleVRF v = deriveVerKeyVRF sk
-    r <- getR
+
+    let r = fromIntegral (bsToNat y) `mod` q
```

Comment:
While we're at it, can we add an extra patch to replace all uses of `bsToNat` by `bytesToNatural` from the `Util` module, since that one has an efficient implementation, and tests.

Reply:
done!